test/decl/protocol/conforms/nscoding.swift

```diff
@@ -8,6 +8,7 @@
 // RUN: %FileCheck --check-prefix=NEGATIVE %s < %t/old.ast
 
 // REQUIRES: objc_interop
+// UNSUPPORTED: OS=iosmac
```

Comment:
Could you provide some explanation for this change? Is it unsupported (which means we need to write a different test that tests this feature), or is it like an XFAIL -- and then we need a tracking bug?

Reply:
This does not need to be in this update, going to remove it. 
```suggestion

```